db/schema.rb

```diff
@@ -141,7 +140,6 @@
 
   add_foreign_key "activities", "activities"
   add_foreign_key "activities", "organisations", column: "extending_organisation_id"
-  add_foreign_key "activities", "organisations", column: "reporting_organisation_id"
```

Comment:
Is this supposed to be here?

Reply:
No! I think I need to clean up my db, this keeps reappearing 😬